### ISSUE TITLE
Radius seems to be ignored when searching by reference

### ISSUE THREAD

**Issue by M4rtinK** [rinigus/osmscout-server#162]
For the example query:
`http://localhost:8553/v1/guide?query=Restaurant&limit=32&radius=2000&search=Brno`
Some restaurants that are ~ 51 km distant are returned even though radius set to 2000 m.

OSM Scout Server version: 0.12.0-1
Map: Czech Republic
NLP: ON
NLP language: cs, en
search all available maps: ON (maps of Czech Republic, Austria, Italy (Centro) and Japan are installed)
use primitive parser: ON

**EDIT:** This could be at least (a partial) fluke - the geocoder likely finds another place called "Brno" & the results are <2000 m from it while I'm using approximate distance from my position for reference.

Using a more detailed reference seems to confirm this theory:

`http://localhost:8553/v1/guide?radius=2000&limit=32&query=Restaurant&search=Kounicova+Brno`

The returned restaurants for this query really should by around that street (eq. less than 2 km from it).

So in short - this might not be a bug after all. :)

**Comment by rinigus** [comment]
That should be indeed the case. The server uses the first search result as an origin.

Closing it, since its a known feature. If you disagree, please reopen :)

**Comment by M4rtinK** [comment]
Nice, thanks! :)

BTW, on OpenRepos I see "Bugfix in guide search" in the 0.13.1 release note - does that mean a fix for this issue is included or is it something else ? :)

**Comment by rinigus** [comment]
Hi,

Yes, the fix is included in 0.13.1 :)

Rinigus

**Comment by M4rtinK** [comment]
Nice! Thanks a lot! :-)

**Comment by rinigus** [comment]
Let me know if it fixed the issue

**Comment by M4rtinK** [comment]
> Let me know if it fixed the issue

Yep - even with a radius of 32000 the closest results are still returned. So thanks a lot yet again! :)
